src/toffy/watcher_callbacks.py

```diff
@@ -141,11 +146,26 @@ def _generate_fov_data(
             time_res=time_res,
         )
 
+        panel_prof = generate_prof_panel(panel)
```

Comment:
I think it would be worth having a flag that controls whether this happens or not.

Reply:
We should add a similar flag to notebook 3b since that also extracts proficient data.